### PR TITLE
refactor metrics api by applying new def of symptom and behavior count

### PR DESCRIPTION
**Pull request by nodestory**
This PR is a follow-up changes in order to bring the new definition of symptom and behavior count to metrics API.

Usually we need to get metrics for both today and yesterday. Before I had to query twice for the two metrics respectively. But actually we can specify the time range which spans two days and let MongoDB aggregate the results by date. One more stage in aggregation, one less db query. This PR adopts this approach.

### CODE REVIEW

File: store/good_behavior_test.go [hunk 1]
```diff
@@ -176,6 +176,85 @@ func (s *BehaviorTestSuite) TestFindNearbyBehaviorReportTimes() {
...
+}
+
+func (s *BehaviorTestSuite) TestGetBehaviorCountForCommunity() {
+	store := NewMongoStore(s.mongoClient, s.testDBName)
+
+	loc := &schema.Location{
+		Longitude: locationBitmark.Coordinates[0],
+		Latitude:  locationBitmark.Coordinates[1],
+	}
+	dist := consts.CORHORT_DISTANCE_RANGE
```
Comment (pieceofr) [inline store/good_behavior_test.go:208]: NEARBY_DISTANCE_RANGE 
I suggest remove CORHORT_DISTANCE_RANGE or delete NEARBY_DISTANCE_RANGE and change  CORHORT_DISTANCE_RANGE = 1000

File: store/good_behavior_test.go [hunk 2]
```diff
@@ -176,6 +176,85 @@ func (s *BehaviorTestSuite) TestFindNearbyBehaviorReportTimes() {
...
+
+	now := time.Date(2020, 5, 25, 12, 0, 0, 0, time.UTC)
+	todayCount, yesterdayCount, err := store.GetBehaviorCount("", loc, dist, now)
+	assert.NoError(s.T(), err)
```
Comment (pieceofr) [inline store/good_behavior_test.go:212]: personally prefer using s.NowError(expected, actual)

Comment (lemonlatte) [reply store/good_behavior_test.go:212]: Yes. As well as, `s.Equal(2, todayCount)` for below

File: store/good_behavior_test.go (hunk 1 above)
Comment (nodestory) [reply store/good_behavior_test.go]: I can handle this in a separate PR.

Comment (pieceofr) [reply store/good_behavior_test.go]: At least change CORHORT_DISTANCE_RANGE = 1000

**Pull request merged**